routes/api.php

```diff
@@ -1,5 +1,8 @@
 <?php
 
+use Rapidez\VatValidation\Http\Controllers\VatController;
+
 Route::middleware('api')->prefix('api')->group(function () {
-    //
+    Route::post('vat-validate', VatController::class)
+        ->middleware('throttle:'.config('rapidez.vat-validation.rate-limit').',1');
```

Comment:
API routes are throttled by default: https://github.com/laravel/laravel/blob/10.x/app/Http/Kernel.php#L43, do we need this specific rate limit?

Reply:
Not necessarily, I had left it in here because the VIES service is already notoriously overwhelmed and I don't necessarily want to make that any worse. I'll take it off for now though.